lib/balance-tx/lib/internal/Internal/Cardano/Write/UTxOAssumptions.hs

```diff
@@ -48,11 +41,3 @@ assumedInputScriptTemplate = \case
     AllKeyPaymentCredentials -> Nothing
     AllByronKeyPaymentCredentials -> Nothing
     AllScriptPaymentCredentialsFrom scriptTemplate _ -> Just scriptTemplate
-
-validateAddress :: UTxOAssumptions -> Address -> Bool
-validateAddress = valid
```

Comment:
Will want this for #3994, but fair enough

Reply:
@Anviking We can definitely revert individual commits if the code they contain is needed in future.